data/etl/movieshowing.py

```diff
@@ -12,38 +18,122 @@ class MovieShowing:
 
     def __init__(self, cinema):
         self.driver = webdriver.PhantomJS()
+        self.driver.set_window_size(1124, 850)  # set browser size
         self.cinema_id, self.cinema_name, self.cinema_url = cinema
 
     def extract_cinema_schedule(self):
         """retrieve one cinema schedule based on the given url,
-        return a list of dictionaries contains """
-        # retrieve title, (type like 3D) and schedule time
-        print(self.cinema_url)
+        return a list of dictionaries of movie titles
+
+        Main logic flow as follow:
+            1. extract_raw
+            2. transform and packaging
+            3. match and store imdb id
+            4. return the data
+        """
+        # self.extract_raw_golden_village()
+        # self.extract_raw_cathay()
```

Comment:
the functionality of this part has been moved to another module. Will be refactored in the next change